addons/website/static/src/builder/website_builder.js

```diff
@@ -61,7 +64,10 @@ export class WebsiteBuilder extends Component {
         const pluginsToRemove = this.props.translation
             ? [...mainEditorPluginsToRemove, ...pluginsBlockedInTranslationMode]
             : mainEditorPluginsToRemove;
-        const mainEditorPlugins = removePlugins([...MAIN_EDITOR_PLUGINS], pluginsToRemove);
+        const mainEditorPlugins = removePlugins(
+            [...MAIN_EDITOR_PLUGINS, ...NO_EMBEDDED_COMPONENTS_FALLBACK_PLUGINS],
```

Comment:
I'm not sure why this line is needed. If they don't use the html field then they should not have the automatic addition of the `NO_EMBEDDED_COMPONENTS_FALLBACK_PLUGINS` since the addition of those is linked to the `embeddedComponents` prop.